versions.html

```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.13.0/">v0.9.0 (release candidate)</a>
```

Comment:
could you please update 0.9.0 to 0.13.0.

Reply:
Oops thanks for the catch!